client/README.md

```diff
@@ -33,7 +33,11 @@ Here is example code that subscribes to updates for a specific Stacks address:
 import { connectWebSocketClient } from '@stacks/blockchain-api-client';
 
 // for testnet, replace with wss://api.testnet.hiro.so/
-const client = await connectWebSocketClient('wss://api.mainnet.hiro.so/');
+const client = await connectWebSocketClient('wss://api.mainnet.hiro.so/', {
+  headers: {
+    'x-hiro-product': 'blockchain-api-client',
+  },
+});
```

Comment:
I don't think this is invalid code, there's no second parameter where headers can be specified. In typescript this won't compile, in javascript this second param will be ignored: 
https://github.com/hirosystems/stacks-blockchain-api/blob/9dc72488bff40de008d4245172259185840fc670/client/src/ws/index.ts#L318-L322

Also it looks like according to the WebSocket spec, clients are not permitted to set request headers, see https://stackoverflow.com/a/4361358/794962

Reply:
There might be a way for us to modify the client code to always include this header instead... not sure if that could be auto-generated though, we might need to do that manually